### DISCUSSION TITLE
Support for correcting horizontal scrolling

### DISCUSSION THREAD

**Discussion by danielkeydigital** [linearmouse/linearmouse#123] [Ideas]
Mice that allow horizontal scrolling by pushing the mouse wheel to the left or right are currently reversed. 
i.e. To scroll to the right I have to push my mouse wheel to the left.

**Comment by lewisl** [comment]
I can't even get the horizontal scroll behavior at all with lean wheel.

I use shift-scroll and reverse orientation.  Works great and is smoother than lean wheel could ever be.  So, no biggee

Just noting that this is likely a broader problem with lean wheel.
